### ISSUE TITLE
cran_packages("varhandle") errors

### ISSUE THREAD

**Issue by maelle** [r-hub/pkgsearch#72]
``` r

pkgsearch::cran_packages("varhandle")
#> Error in data.frame(stringsAsFactors = FALSE, type = type, package = names(deps[type][[1]]), : arguments imply differing number of rows: 1, 0
pkgsearch::cran_package("varhandle")
#> CRAN package varhandle 2.0.4, 15 days ago
#> Title: Functions for Robust Variable Handling
#> Maintainer: Mehrad Mahmoudian <m.mahmoudian@gmail.com>
#> Author: Mehrad Mahmoudian [aut, cre]
#> BugReports:
#>     https://bitbucket.org/mehrad_mahmoudian/varhandle/issues
#> Date: 2019-10-21
#> Date/Publication: 2019-10-21 17:30:02 UTC
#> Depends: R (>= 3.0.1)
#> Description: Variables are the fundamental parts of each
#>     programming language but handling them efficiently might be
#>     frustrating for programmers. This package contains some
#>     functions to help user (especially data explorers) to make
#>     more sense of their variables and take the most out of
#>     variables and hardware resources. These functions are written,
#>     collected and crafted over 7 years of experience in
#>     statistical data analysis on high-dimensional data, and for
#>     each of them there was a need. Functions in this package are
#>     suppose to be efficient and easy to use, hence they will be
#>     frequently updated to make them more convenient.
#> Imports: utils (*), graphics (*)
#> License: GPL (>= 2)
#> MD5sum: 435dd8b0edb763d1deb5d379cd384195
#> NeedsCompilation: no
#> Packaged: 2019-10-21 17:12:21 UTC; mehrad
#> releases:
#> Repository: CRAN
#> Suggests: ()
#> URL: https://bitbucket.org/mehrad_mahmoudian/varhandle
```

<sup>Created on 2019-11-05 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by maelle** [comment]
Found the bug whilst trying to run

```r
trending <- pkgsearch::cran_trending()
meta <- pkgsearch::cran_packages(trending[["package"]])
```

**Comment by maelle** [comment]
Interesting https://github.com/cran/varhandle/blob/e30ec260ec3a88d01bebf6b3e135165597e54614/DESCRIPTION#L11

**Comment by maelle** [comment]
Fixed by #73
